### DISCUSSION TITLE
The meaning of <> </>

### DISCUSSION THREAD

**Discussion by jpeiyi9** [vercel/next.js#33380] [Help] [answered]
I find this syntax at https://nextjs.org/learn/basics/navigate-between-pages/link-component#:~:text=function%20FirstPost()%20%7B%0A%20%20return%20(-,%3C%3E,-%3Ch1%3EFirst%20Post Anyone knows what the meaning of it?

**Comment by mattcarlotta** [comment, marked as answer]
They’re [shorthand](https://reactjs.org/docs/fragments.html#short-syntax) JSX for a React [Fragment](https://reactjs.org/docs/fragments.html). However, unlike a `<Fragment>`, they can’t contain a `key` prop.

**Reply by jpeiyi9** [reply]
It's helpful. Thanks!
